### PR TITLE
Created docs folder

### PR DESCRIPTION
**Pull request by Trinovantes**
- Changed README to general info about the project
- Moved installation instructions to docs/INSTALL.md

TODO: Need to clarify `docs/INSTALL.md` on what computer to run the commands on. We may just want to add a single bash script (with user defined variables at the top) to run all the commands. The important thing is to minimize the amount of commands we need to run to provision a brand new Ubuntu server.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,113 +1,49 @@
...
-```
-
-Inside the Postgres terminal enter:
-
-```
-\password sample_db_password
-```
-
-Once prompted for the password enter `sample_db_password` again and then exit with `Ctrl+D`. Finally enter:
+The Sana Protocol is an XML specification for defining medical procedures. Basically, Sana is to HTML as medical documents is to web pages. These Sana procedures are then downloaded by nurses and doctors onto their phones with the Sana mobile application. These mobile applications will then parse the Sana document and generate a questionaire "procedure" that the nurse or doctor can ask their patients. An example document is shown below:
```
Comment (Kapin) [inline README.md]: as medical documents "are"

Comment (Kapin) [inline README.md]: Less thens. "These Sana procedures are downloaded..."

File: README.md [hunk 2]
```diff
@@ -1,113 +1,49 @@
...
+
+```
```
Comment (Kapin) [inline README.md]: Define xml syntax highlighting?

**Comment by Kapin** [comment]
Other than grammar, LGTM

File: README.md (hunk 2 above)
Comment (Trinovantes) [inline README.md]: Huh?

Comment (Kapin) [inline README.md]: Like make sure it actually highlights the procedure xml properly

Comment (Trinovantes) [inline README.md]: It's not symmetrical :(

```xml
<Procedure title="Neuro Module" author="Partners For Care">
  <Page>
    <Element type="MULTI_SELECT" id="1" 
      concept="NEURO SYMPTOM" 
      question="What are the symptoms?" 
      answer=""
      choices=“Dizziness,Fainting,Headaches,Other complaints"/>
  </Page>
  <Page>
		<ShowIf>
			<Criteria type="EQUALS" id="1" value="Fainting"/>
		</ShowIf>
		<Element type="ENTRY" 
		concept="UNCONSCIOUSNESS" id="2" 
		question="How long was the patient unconscious?" 
		answer=""/>
  </Page>
  <Page>
		<ShowIf>
			<Criteria type="EQUALS" id="1" value="Other complaints"/>
		</ShowIf>
		<Element type="ENTRY" 
		concept="OTHER COMPLAINTS" id="4" 
		question="Are there any other complaints?"
		answer=""/>
  </Page>
  ...
</Procedure>
```

Comment (Kapin) [inline README.md]: good point. Ignore me

**Pull request merged**